ports/mapnik/portfile.cmake

```
@@ -0,0 +1,111 @@
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO mapnik/mapnik
+    REF 737e7a69c2679b33093e1ebf0b4c3fa34ae36878
+    SHA512 ff57fdb27eff8a83519601a30428f6979b042e8029484da8a3605a75d58120eaaaf57dfb456cb19d9078cfbdca3bd1f8fc38cc31c61cde3cd037ef246853ca73
+    HEAD_REF master
+    PATCHES
+      "use-proj.patch"

```

```suggestion
    PATCHES
        use-proj.patch
```
The spaces should be four here. And it's unnecessary to add quotes for the patch.